src/components/content/rich-text/rich-text.tsx

```diff
@@ -57,7 +57,7 @@ export const ContentfulRichText = ({ data }: ContentfulRichTextProps) => {
   /**
    * Filter out footnotes from content references
    */
-  const footnotes: ContentfulFootnoteReference[] = data.references.filter(
+  const footnotes: ContentfulFootnoteReference[] = data.references?.filter(
```

Comment:
`?` is a code smell and should be avoided. Especially with lists, try to make sure that they are never `null` or `undefined` but always empty. 

If you change this line here to 
```
data.references?.filter(...) | | []
```
you can remove all further `?` checks here in this file.

Reply:
Much better 👍